.circleci/config.yml

```diff
@@ -48,8 +66,12 @@ references:
 
   android_config: &android_config
     working_directory: *workspace
+    environment:
+#        GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xms1536m -Xmx4096m -XX:+HeapDumpOnOutOfMemoryError"'
+#        GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xms1536m -Xmx4096m -XX:+HeapDumpOnOutOfMemoryError" -Dorg.gradle.parallel=true'
+        GRADLE_OPTS: '-Dorg.gradle.parallel=true'
     docker:
-      - image: circleci/android:api-26-alpha
+      - image: circleci/android:api-27-alpha
```

Comment:
It'd be nice to have a comment inline saying you used this to avoid an update.

Reply:
I did comment on commit #840562 message: a select android api 27 machine image to avoid an unnecessary platform update

Is an in-line comment desired as well?

Reply:
Yes, I saw that comment, but it's the kind of thing that might be confusing in the future and it's nice not to go back in history. I don't feel strongly about it though.

Reply:
Ok, I'll add a comment then